nixos/modules/services/networking/routinator.nix

```
@@ -0,0 +1,71 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.routinator;
+  settingsFormat = pkgs.formats.toml { };
+in
+{
+  options.services.routinator = {
+    enable = mkEnableOption "Routinator 3000 is free, open-source RPKI Relying Party software made by NLnet Labs.";
+
```

It is customary to expose the package as an option. That also means you should use only reference `cfg.package`.

```suggestion

  package = mkPackageOption pkgs "routinator" {} ;
  
```